aim/tools/cli/commands/manager.py

```diff
@@ -248,12 +274,42 @@ def load_domains(ctx, replace, enforce):
                           for x in manager.find(aim_ctx,
                                                 resource.PhysicalDomain)]
             all_epgs = manager.find(aim_ctx, resource.EndpointGroup)
+            if mappings:
+                host_mappings = {}
```

Comment:
Nit: do we need this dict? Two variables would be just fine

Reply:
Sure -- will change.